natives.json

```
@@ -99718,17 +99718,17 @@
 			"build": "323"
 		},
 		"0xE44A982368A4AF23": {
-			"name": "_0xE44A982368A4AF23",
+			"name": "COPY_VEHICLE_DAMAGES",
 			"jhash": "0xBAE491C7",
-			"comment": "",
+			"comment": "Copy the damage like broken bumpers and broken lights from a vehicle and paste then on another vehicle.",

```

Please fix the comment's grammar, e.g.
```suggestion
			"comment": "Copies sourceVehicle's damage (broken bumpers, broken lights, etc.) to targetVehicle.",
```
This is also more clear and shorter.

---

Changes maded. I tried to rebase and merge these 3 commits into a single one, but i don't know how to do that, sorry.